src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -351,7 +352,7 @@ private void recoverCommitlogAndCompleteSetup() {
                 && StorageService.instance.hasNonTransientError(StorageServiceMBean.NonTransientError.COMMIT_LOG_CORRUPTION))
             {
                 logger.error("Failed to recover from commitlog corruption due to some non transient errors: {}",
-                             StorageService.instance.getNonTransientErrors());
+                             UnsafeArg.of("commitLogRecoverError", StorageService.instance.getNonTransientErrors()));
```

Comment:
nit: should maybe be an optional plural, since I think this can contain multiple errors